apps/react-18-tests-v9/package.json

```diff
@@ -19,14 +19,17 @@
     "@fluentui/scripts-tasks": "*"
   },
   "dependencies": {
+    "@cypress/react": "9.0.0",
     "@fluentui/react-components": "*",
-    "@testing-library/dom": "^10.1.0",
-    "@testing-library/react": "^16.0.0",
-    "@testing-library/user-event": "^14.5.2",
-    "@types/react": "18.0.14",
-    "@types/react-dom": "18.0.6",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
+    "@testing-library/dom": "^10.4.0",
```

Comment:
with this new approach. is testing library and friends even used ? if not please remove

Reply:
It is, because the app contains unit tests that I didn't remove. They can be changed in a follow-up PR, as they are not comprehensive as the E2E tests.